third_party/move/move-compiler-v2/tests/bytecode-generator/freeze_mut_ref.exp

```diff
@@ -292,3 +292,16 @@ fun freeze_mut_ref::t8($t0: bool, $t1: &mut freeze_mut_ref::S, $t2: &freeze_mut_
   6: label L2
   7: return ()
 }
+
+
+Diagnostics:
```

Comment:
Thought about fixing the error in the test file, but it changes the test a bit too much for my liking, so left it in.

Reply:
I note that v1 gives a bunch of warnings about "unused assignment or binding" for this file (58x2, 66, 69, 74x2).  Are we suppressing such errors on this file or are we just not hitting it yet due to phase ordering?

Btw, V1 doesn't have the borrow issue below.